src/docs/product/integrations/notification-incidents/threads/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Threads
+sidebar_order: 1
+description: "Learn more about Sentry's Threads integration and how you can effortlessly send alerts from Sentry to Threads."
+---
+
+The Threads Sentry integration allows you to configure Threads as a destination for issue and metric alerts. Threads will keep you updated on the status of the Sentry issue, and let you know when a metric alert has been triggered and resolved.
+
+This integration is maintained and supported by the company that created it. For more details, check out our [Integration Platform documentation](/product/integrations/integration-platform/).
+
+## Install and Configure
+
+<Note>
+
+Sentry owner, manager, or admin permissions are required to install this integration.
+
+Threads **won't** work with self-hosted Sentry.
+
+</Note>
+
+To add the Threads integration, navigate to **Settings > Integrations > Threads** and click "Install".
+
+![](threads-install.png)
+
+### Alerts
+
+The Threads Sentry integration supports [issue alerts](/product/alerts/alert-types/#issue-alerts) and [metric alerts](/product/alerts/alert-types/#metric-alerts). When selecting the [action](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions), choose "Notify Integration" and then "Threads" for issue alerts, or select "Threads" from the action dropdown for metric alerts. Finally, choose the channel to be notified in:
+
+![](threads-rule.png)
+
+A Threads notification will be created automatically when the alert is triggered.
+
+
```

Comment:
I _kinda_ feel like there should be a heading here but I'm not sure, and if there is, I'm unsure what to call it.

Reply:
This section seems to be talking about issues rather than issue alerts. But then you go back to talking about metric alerts. Maybe the metric alert content should come first as part of the Alert Rules section, and the part about issues should be under its own H2 heading of some sort. I'm thinking "Issue Statuses" or something.